addons/binding/org.openhab.binding.rfxcom.test/src/test/java/org/openhab/binding/rfxcom/internal/messages/RFXComBarometricMessageTest.java

```diff
@@ -1,6 +1,6 @@
 /**
- * Copyright (c) 2010-2016 by the respective copyright holders.
- * <p>
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
```

Comment:
Should be removed on this PR as this is not related to Neato binding. If valid fix, separate PR should be done.